.github/workflows/test-code.yml

```diff
@@ -2,6 +2,11 @@ name: ZITADEL PR
 
 on:
   pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - 'guides/**'
+      - 'tools/**'
```

Comment:
@adlerhurst changes in this folder have an impact on the build / code, right?

Reply:
yes. The binaries installed from this mod file may change behaviours

Reply:
not ignoring the directory anymore https://github.com/zitadel/zitadel/pull/4152/commits/2858f5c23989384e082cd8f9255f133027612fed